client/src/components/Conversations/Convo.tsx

```diff
@@ -48,31 +46,20 @@ export default function Conversation({ conversation, retainView, toggleNav, isLa
 
     // set conversation to the new conversation
     if (conversation?.endpoint === EModelEndpoint.gptPlugins) {
-      let lastSelectedTools = [];
-      try {
-        lastSelectedTools = JSON.parse(localStorage.getItem('lastSelectedTools') ?? '') ?? [];
-      } catch (e) {
-        // console.error(e);
-      }
+      const lastSelectedTools = JSON.parse(localStorage.getItem('lastSelectedTools') ?? '') ?? [];
       navigateToConvo({ ...conversation, tools: lastSelectedTools });
     } else {
       navigateToConvo(conversation);
     }
   };
 
-  const renameHandler = (e: MouseEvent<HTMLButtonElement>) => {
+  const renameHandler = (e) => {
     e.preventDefault();
     setTitleInput(title);
     setRenaming(true);
-    setTimeout(() => {
```

Comment:
why remove the focus?

Reply:
I thought it would be interesting for the user to be able to click elsewhere on the screen without losing the title changes 😅